source/embedded-shell.txt

```diff
@@ -66,6 +66,31 @@ an :ref:`aggregation pipeline <aggregation-pipeline>`:
         { $match: { "last_name": "Smith" } } // press Shift + Enter
       ] )                                    // Press Enter
 
+Disable the Embedded MongoDB Shell
+----------------------------------
+
+You can disable the embedded MongoDB shell in |compass-short| to restrict 
```

Comment:
Im a little confused by " to restrict users from running unauthorized commands on :binary:`mongosh`."

This setting is the client specific one that affects only the current users Compass client.
If a MongoDB admin were to facilitate the install of Compass with a global config file this could be true, but its not obvious who the "you" is in this context.

I would argue its also unclear in our current config page: https://www.mongodb.com/docs/compass/current/config-file/. 

Perhaps this could be shortened to:

"You can disable the embedded MongoDB shell in |compass-short|"

Reply:
Ah that's a great point. Updated.